documentation/slate/source/includes/journey_planning.md

```diff
@@ -1,17 +1,24 @@
-<a name="journey_planning"></a>Journey planning
+<a name="journey_planning"></a>![Journey planning](/images/journeys.png)Journey planning
 ===============================================
 
--   **[Journeys](#journeys)** : Compute journeys
+The multi-modal itinerary feature allows you to compute best routes from point A to point B 
+using all available means of travel, including: bus, train, subway, bike, public bike, walking, car, etc.
+This function returns a roadmap with specific instructions for a route based on available information, 
+such as: time of departure and arrival, journey time, possible modes of transport, and walking distance.
 
-| url | Result |
-|------------------------------------------|-------------------------------------|
-| `get` /coverage/*resource_path*/journeys | List of journeys                    |
-| `get` /journeys                          | List of journeys                    |
+In order to compute a journey, you may have to use these APIs (click on them for details):
 
--   **[Places](#places)** : autocomplete for geographical data
+-   **[Places](#places)** : autocomplete on geographical data to find the departure and destination points from an input text.
 
 | url | Result |
 |------------------------------------------------|-------------------------------------|
-| `get` /coverage/places                         | List of geographical objects        |
+| `/coverage/places`                             | List of geographical objects        |
+
+-   **[Journeys](#journeys)** : compute journeys from points or coordinates
```

Comment:
maybe it would be better to put places with journeys no ?

you usually need to use places before, but Journeys is more important there no ?